exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusMetricReaderTest.java

```diff
@@ -1055,26 +1031,28 @@ void otelScopeLabelsOnly() throws IOException {
     assertThat(toOpenMetrics(reader.collect())).isEqualTo(expected);
   }
 
-  @SuppressWarnings("resource")
-  @Test
-  void addResourceAttributesWorks() throws IOException {
-    PrometheusMetricReader reader =
-        new PrometheusMetricReader(/* allowedResourceAttributesFilter= */ Predicates.is("cluster"));
+  private void addCounter(PrometheusMetricReader reader, ResourceBuilder builder) {
     Meter meter =
         SdkMeterProvider.builder()
             .setClock(testClock)
             .registerMetricReader(reader)
-            .setResource(
-                Resource.getDefault().toBuilder()
-                    .put("cluster", "my.cluster")
-                    .put("telemetry.sdk.version", "1.x.x")
-                    .build())
+            .setResource(builder.put("telemetry.sdk.version", "1.x.x").build())
             .build()
             .meterBuilder("test-scope")
             .setInstrumentationVersion("a.b.c")
             .build();
     LongCounter counter = meter.counterBuilder("test.count").build();
     counter.add(1);
+  }
+
+  @SuppressWarnings("resource")
```

Comment:
I know you didn't write this but this suppression is redundant since this warning is already suppressed at the class level